src/nomad_simulations/schema_packages/properties/thermodynamics.py

```diff
@@ -0,0 +1,289 @@
+    """
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class InternalEnergy(BaseEnergy):
+    """
+    Physical property section describing the internal energy (i.e., heat added - work) for a (sub)system.
+    """
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class Enthalpy(BaseEnergy):
+    """
+    Physical property section describing the enthalpy (i.e. internal_energy + pressure * volume.) of a (sub)system.
+    """
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class Entropy(PhysicalProperty):
+    """
+    Physical property section describing the entropy of a (sub)system.
+    """
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule / kelvin',
+        description="""
+        The value of the entropy.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class GibbsFreeEnergy(BaseEnergy):
+    """
+    Physical property section describing the Gibbs free energy (i.e., enthalpy - temperature * entropy) of a (sub)system.
+    """
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class HelmholtzFreeEnergy(BaseEnergy):
+    """
+    Physical property section describing the Gibbs free energy (i.e., internal_energy - temperature * entropy) of a (sub)system.
+    """
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class ChemicalPotential(BaseEnergy):
+    """
+    Free energy cost of adding or extracting a particle from a thermodynamic system.
+    """
+
+    # ! implement `iri` and `rank` as part of `m_def = Section()`
+
+    iri = 'http://fairmat-nfdi.eu/taxonomy/ChemicalPotential'
+
+    def __init__(
+        self, m_def: 'Section' = None, m_context: 'Context' = None, **kwargs
+    ) -> None:
+        super().__init__(m_def, m_context, **kwargs)
+        self.rank = []
+        self.name = self.m_def.name
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class HeatCapacityCV(PhysicalProperty):
+    """
+    Physical property section describing the heat capacity at constant volume for a (sub)system.
+    """
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule / kelvin',
+        description="""
+        The value of the heat capacity.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class HeatCapacityCP(PhysicalProperty):
```

Comment:
do we need both properties, Cp and Cv? If variables is set anyways, then that should classify the property already.

This was my thought when defining `HeatCapacity` on the taxonomy as well.

Reply:
ok, good, I also had this question, I just forgot to put a comment. I will make it consistent with the taxonomy then

Reply:
I noticed that here you do not inherit from `PhysicalContribution`.
Tbh, I don't have a use case that warrants this, but maybe leave a comment in case you need to include it later on.

Reply:
> do we need both properties, Cp and Cv? If variables is set anyways, then that should classify the property already.

Good point, but note that you are now assuming that all non-variables are implicitly constants. Or is this only for dual variables like P and V?  I'm fine if you want to put this writing, but note that in the math notation these are defined separately. This is for the same reason that you have the distinction between (total) derivatives and partial derivatives.